pkg/kubelet/apis/config/types.go

```diff
@@ -478,6 +478,11 @@ type KubeletConfiguration struct {
 	// If not specified the default value is ContainerRuntimeEndpoint
 	// +optional
 	ImageServiceEndpoint string
+
+	// PodresourcesLimits specifies the rate limits for the node-local podresources API endpoint
+	// +featureGate=KubeletPodResources
+	// +optional
+	PodresourcesLimits RateLimitConfiguration
```

Comment:
`Podresources` or `PodResources`? We seem to have a mix of both, but we should be consistent. I think `PodResources` is probably more appropriate since it's two separate words.

Reply:
Good point. I will update the PR.

Reply:
agree with `PodResources` as the field name and `podResources` as the serialized json name (in v1beta1/types)

Reply:
Thank you both. Fixed as suggested.